pkg/curatedpackages/regional_registry.go

```diff
@@ -17,7 +17,7 @@ import (
 const (
 	devRegionalECR       string = "067575901363.dkr.ecr.us-west-2.amazonaws.com"
 	devRegionalPublicECR string = "public.ecr.aws/x3k6m8v0"
-	stagingRegionalECR   string = "067575901363.dkr.ecr.us-west-2.amazonaws.com"
+	stagingRegionalECR   string = "724423470321.dkr.ecr.us-west-2.amazonaws.com"
```

Comment:
Why don't we add a stagingRegionalECRMap to map all regions to the corresponding staging account private ECR similar to prodRegionalECRMap?

Reply:
We are not going to run staging tests in every region, we just need one place where we can get staging artifacts from. So PDX gamma would do.